include/wx/textctrl.h

```diff
@@ -680,6 +685,17 @@ class WXDLLIMPEXP_CORE wxTextAreaBase
     virtual wxString GetValue() const = 0;
     virtual void SetValue(const wxString& value) = 0;
 
+    // overridden implementations are required to actually handle RTF
+    virtual wxString GetRTFValue() const
+    {
+        wxFAIL_MSG("GetRTFValue() should only be used with rich, multiline controls.");
```

Comment:
Is this message really correct? I think it should say "not implemented for the current platform".

I'd also move this (and the function below) into `src/common/textcmn.cpp`, there doesn't seem to be any good reason to define them inline.

Reply:
Got it.